roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -748,6 +748,29 @@ class OpenShiftCLIError(Exception):
     pass
 
 
+def locate_oc_binary():
```

Comment:
@ashcrow Utils is defined in src/lib/base.py, which is where `locate_oc_binary()` is already defined.